libs/algo/odc/algo/io.py

```diff
@@ -307,7 +307,7 @@ def fuser(xx: xr.Dataset) -> xr.Dataset:
     pad = kw.pop("pad", None)
     if pad is None:
         if filters is not None:
-            pad = max(*filters.values())
+            pad = max(list(zip(*filters))[1])
```

Comment:
filters is a tuple not a dict anymore so I converted it to list object of operations and radius first and then get a max from radius list. 

So not changing the implementation but fixed due to change in filters definition.

Reply:
Ok, I trust you. It's pretty hard to read, but if you're happy I'm happy!

Reply:
I know it is hart to iterate through tuple but this works :)